src/components/Form.js

```diff
@@ -0,0 +1,37 @@
+const Form = (props) => {
+    // Destructuring props
+    const { selectOptions, callSpecificEndpoint, userChoice, setUserChoice } = props;
```

Comment:
consider destructuring like this to dry up your code/you won't even have to use props dot notation:

const Form = ( { selectOptions, callSpecificEndpoint, userChoice, setUserChoice } ) {
    
}